docs/pages/contributing.mdx

```diff
@@ -59,6 +59,32 @@ npm run docs
 You can now access the documentation site locally. Changes to the docs will hot
 reload the site.
 
+### Documentation Goals
+
+As an organization, we've established a few goals for our documentation in order
+to keep them accessible, friendly, and useful.
+
+- Docs should be friendly and human
+
+Our documentation should be accessible to everyone, regardless of their skill
+level with React. This means we should try to write documentation that someone
+totally new to React can get started with.
+
+It also means that we want the voice of our docs to be simple and casual. We
```

Comment:
Maybe "casual and straightforward" would work better here.

Reply:
Great suggestion @aaronmcadam!